src/PointwiseFunctions/GeneralRelativity/Psi4Real.hpp

```diff
@@ -0,0 +1,94 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/CoordinateMaps/Tags.hpp"
+#include "Domain/ElementMap.hpp"
+#include "Domain/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Psi4.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace gr {
+
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the real portion of the Newman Penrose quantity \f$\Psi_4\f$
+ * using  \f$\Psi_4[Real] = -0.5*U^{8+}_{ij}*(x^ix^j - y^iy^j)\f$.
+ */
+template <typename Frame, typename RealDataType>
+void psi_4_real(
+    const gsl::not_null<Scalar<RealDataType>*> psi_4_real_result,
+    const tnsr::ii<RealDataType, 3, Frame>& spatial_ricci,
+    const tnsr::ii<RealDataType, 3, Frame>& extrinsic_curvature,
+    const tnsr::ijj<RealDataType, 3, Frame>&
+        cov_deriv_extrinsic_curvature,
+    const tnsr::ii<RealDataType, 3, Frame>& spatial_metric,
+    const tnsr::II<RealDataType, 3, Frame>& inverse_spatial_metric,
+    const tnsr::I<RealDataType, 3, Frame>& inertial_coords) {
+  const auto psi_4_complex = psi_4<ComplexDataVector>(
+      spatial_ricci, extrinsic_curvature, cov_deriv_extrinsic_curvature,
+      spatial_metric, inverse_spatial_metric, inertial_coords);
+  get(*psi_4_real_result) = real(get(psi_4_complex));
+}
+
+template <typename Frame, typename RealDataType>
+Scalar<RealDataType> psi_4_real(
+    const tnsr::ii<RealDataType, 3, Frame>& spatial_ricci,
+    const tnsr::ii<RealDataType, 3, Frame>& extrinsic_curvature,
+    const tnsr::ijj<RealDataType, 3, Frame>&
+        cov_deriv_extrinsic_curvature,
+    const tnsr::ii<RealDataType, 3, Frame>& spatial_metric,
+    const tnsr::II<RealDataType, 3, Frame>& inverse_spatial_metric,
+    const tnsr::I<RealDataType, 3, Frame>& inertial_coords) {
+  auto psi_4_real_result = make_with_value<Scalar<RealDataType>>(
+      get<0, 0>(inverse_spatial_metric), 0.0);
+  psi_4_real(make_not_null(&psi_4_real_result), spatial_ricci,
+            extrinsic_curvature, cov_deriv_extrinsic_curvature, spatial_metric,
+            inverse_spatial_metric, inertial_coords);
+  return psi_4_real_result;
+}
+
+
+namespace Tags {
+/// Computes the real portion of the Newman Penrose quantity \f$\Psi_4\f$ using
```

Comment:
I would say real part, not real portion

Reply:
forgot to put this change in my fixup, will put it in for the next comments I get